.circleci/config.yml

```diff
@@ -58,17 +58,17 @@ commands:
     steps:
       - run:
           name: Decode license in env variable
-          command: echo $BASE64_LICENSE | base64 --decode >> Unity_v2019.x.ulf
+          command: echo $UNITY_LICENSE | base64 --decode >> Unity.x.ulf
```

Comment:
I used a separate env var because I didn't want to modify the existing one, that way existing tests don't break